src/stack.js

```diff
@@ -1,5 +1,16 @@
-'use strict'
-
 export default class Stack {
-  // your code here
+  constructor() {
+    this.elements = []
+    this.size = 0
+  }
+
+  push = (element) => this.elements[this.size++] = element
```

Comment:
I strongly recommend the use of explicit blocks when writing code - this format can be hard to obtain an immediate understanding of, since it is so ... condensed.  Using explicit blocks helps improve readability.

Stylistically, I prefer not to use fat arrow function declarations inside of a class, but many disagree <shrug>:
```js
  push( element ) {
    this.elements[ this.size++ ] = element
  }
```